prestoadmin/util/local_config_util.py

```diff
@@ -27,7 +27,7 @@ def get_config_directory():
 def get_log_directory():
     config_directory = os.environ.get(LOG_DIR_ENV_VARIABLE)
     if not config_directory:
-        config_directory = DEFAULT_PRESTOADMIN_LOG_DIR
```

Comment:
why did you remove this env variable from constants.py instead of just updating the default there (like you have the DEFAULT_LOCAL_CONF_DIR)?